packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -85,6 +91,14 @@ export const Product = ({
     error: false,
   });
 
+  // This effect is to simulate a componentDidMount.
+  useEffect(() => {
+    if (config.metrics.flow.enabled) {
+      FlowMetrics.logLoadedEvent('product', config, getScreenInfo);
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What deps are getting flagged by eslint to require the disabled rule? Looks like `config`, `FlowMetrics`, and `getScreenInfo` - those should be okay to list here. Should mean that this effect gets run on initial render or whenever one of those deps changes (which should basically be never)